tests/test_tweets.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from prefect import flow
+
+from prefect_twitter.tweets import update_status
+
+
+def test_update_status(twitter_credentials):
+    status = "tweeting prefection!"
+
+    @flow
+    def test_flow():
+        media = update_status(twitter_credentials, status=status)
+        return media
+
+    assert test_flow().result().result() == status
+
+
+def test_update_status_missing(twitter_credentials):
+    @flow
+    def test_flow():
+        media = update_status(twitter_credentials)
+        return media
+
+    with pytest.raises(ValueError):
```

Comment:
Can you also include a `match` arg to assert for the message in the `ValueError`? https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions